### DISCUSSION TITLE
Initial set up ... not 'hearing' birds

### DISCUSSION THREAD

**Discussion by Robtlord** [mcguirepr89/BirdNET-Pi#1016] [Q&A]
I thought I posted this a couple days ago but I must have messed up: I have the R-Pi set up with BirdNET and USB microphone on a 6 ft USB extension. Microphone on exterior wall. It has recorded Black Cap Chickadee  but I can clearly hear a Barred Owl hooting and it does not hear. This also happened with a flock of Gold Finches on our sunflowers...chirping away but not heard.  When I played back the Chickadee I could hear it but also a loud hum interference. I don't have experience with sussing this out so looking for some advice as to where to start.

**Comment by waynecarnold** [comment]
Well for what it's worth I will give you what I have done, maybe others can give more insight. I have basically the same setup as you. The Pi is in my shed and the USB microphone on the end of a long cable is outside just under the eaves. At first I was getting very few birds even though when sitting outside, I could hear many. I had to play the the confidence level. The default was not working for me at all unless the bird sang right in my backyard loudly. I have my confidence level set at 0.35 I have played with it and have toyed mainly from 0.27-0.35. Any higher and I miss too many birds that I can hear clearly.

I set the Species Occurence Frequency Threshold to 0.001 which was the only way I could get some birds included (like the barred owl) that is in my area. This setting seems to be working well. The default was just leaving out too many birds and I do not want to miss these rarer ones that I know are around and can hear from time to time.

I left the privacy threshold at the default, I do not want to pick up any conversations. My sigmoid sensitivity is at 1.25. I have raised it in the past and seemed to help pick up some fainter sounds but too many false positives as well so put it back to this level (not really sure exactly what this is doing but is working none the less for me at this setting).

Not sure what can be causing the hum. I guess I would just make sure everything is plugged in tightly. I would plug the mic directly into the Pi and see if the hum is still there. If it is gone, but back when you use the long USB cable extension I would replace it.

Finally what I did when starting out is to play back bird sounds from my phone using the ebird app and making sure it picked it up. I walked further into my yard to see how well it can still pick it up. Not the greatest at times if background noises but works pretty good. 

Oh yes, I also wrapped some foam around my mic to help reduce the wind sound. You know the same material that people use on voice microphones to clean up the background white noise, etc.

Hope this helps somewhat. interested to hear what others say.

**Reply by Awku1961** [reply]
I would guess the hum comes from the power supply. Try to operate your Raspi with an USB Powerpack, if the hum is gone, you have a path to solve that issue.

**Comment by Robtlord** [comment]
Very much appreciate your insights and suggestions. That is exactly what I needed. I won't give up! I will make the advised adjustments and see what happens. I very much like the idea of using my Merlin app to test the accuracy. I had another late night thought... I'll try today: I realized my microphone on the exterior wall is pretty close to an in-wall stereo speaker on the inside. I wonder if the speaker magnet is causing interference? Will start prodding things today! Again thanks.

**Comment by Robtlord** [comment]
Oddly, I played the barred owl recording from the Merlin app 2 feet away from the microphone on 2 occasions and the Birdnet did not respond. The gold finches have been picked up. I moved the mic 3 feet away from where it had been (somewhat near an internal stereo wall speaker, just to be on the safe side)

**Reply by waynecarnold** [reply]
check your Species Occurence Frequency Threshold (in tools section). Unless it is set to 0.001 the barred owl is heard but not uploaded as it is ignored. You can use the tool included to view the species included. The barred owl is way down towards 0.001+ so the default setting will ignore it.

**Comment by Robtlord** [comment]
Again thanks... I have just done so, and will see soon. Also may I ask: the sound card settings is set to 2 channels. I can not find in the microphone literature how many it has (up to 32?!)

**Reply by waynecarnold** [reply]
Don't know that one. I just left mine at default.

**Reply by jflyons** [reply]
Really helpful seeing the discussion from recent days involving Species Occurrence Frequency Threshold and the Species Preview List. The supporting materials (In-depth technical write-up) linked from the Model section of Basic Settings is also helpful. 

My regular use of the new model during 2023 has been very rewarding, but recently when I needed to re-install after needing to replace my hardware, I started to look a little more critically at some of my results.

Tracking the Daily Log while seeing what resulted (or didn't) as far as detections go, I had two questions - first (and lingering from previous install), why did my system not "hear" common species (American Crow, Song Sparrow) most of the time when I could quite clearly hear them, even while indoors and listening through the closed window at times. Daily Log rarely showing them at all let alone at the proper threshold. Then other species (primary example Western Screech Owl) which show on the Daily Log, high enough to be detected, but not ever being reported. While question one remains unanswered for the most part, the second one has converged on being solved - getting a low enough threshold setting to include the otherwise excluded species).

The fact that the data is bad* does not stand in the way of the fact that the Western Screech-Owl's .00126 frequency exceeds the bar if it's set to .001 :) and I was able to set it to that and immediately started picking up detections.

(*WESO really is #234 on our local Week 37 eBird results as reported in the Preview Species List? No way. Among other impossibilities, that "score" puts it beneath four other species of owls!)

**Comment by Robtlord** [comment]
I put the Species Occurrence Frequency down to 0.001 and retried the owl recording 2 ft from microphone. No response. Next up the usb cord.

**Reply by waynecarnold** [reply]
Keep us posted.

**Comment by Robtlord** [comment]
Being new to this everything is a learning moment... trouble is though I find interesting things out I don't know if they are the correct set up! Ha ! So my mic is working though oddly to me.,about a full minute latency. With the Live Mic on I went out with the laptop and talked next to mic. Heard nothing on my laptop speaker so went inside and a minute later I hear myself.... is that normal?

**Reply by waynecarnold** [reply]
Never tried that, I think I would have noticed if that long a delay. FYI here is part of my  sysinfo from the tools which shows how busy the device is, etc. Not sure what it all means but perhaps you can compare. I would think if the device is busy, has less memory, etc then the lag would be there.
![Screenshot 2023-09-04 204342](https://github.com/mcguirepr89/BirdNET-Pi/assets/136210780/afefc4d9-7d8c-4efe-a509-1a0baf809d95)

**Reply by Robtlord** [reply]
Thanks on that I did check and am similarly set up. The one thing I do not get is why I can hear geese honking overhead or a few Barred owls having a very loud conversation for 20 minutes and I see no recognition of that.     Here is my microphone info:  [USB MIC with Hi-Res Sound Card] – Cubilux USB Microphone records crystal sound with built-in Hi-Res Chipset (DAC: 96 KHz/24-bit; ADC: 48 KHz/16-bit) and high sensitivity MIC unit. Ideal for vlog, singing, livestreaming, game chat, video meeting, interviewing etc. Omnidirectional.

**Reply by waynecarnold** [reply]
starting to get beyond my knowledge, lets hope someone else can provide insight. All I can think of is, if you play say the barred owl from a bird app right into the mic so no mistaking it and loudness not an issue and it does not pick it up and all the settings are in place something is wrong and I would perhaps try to reinstall. BUT this is just a guess on the best approach. 

One final thought, make sure you are using the correct model (see image).
![Screenshot 2023-09-05 212357](https://github.com/mcguirepr89/BirdNET-Pi/assets/136210780/c53b54c9-f802-4bcc-88a9-2eacfe957ef0)

**Reply by waynecarnold** [reply]
Also if any human sounds (and sometimes that includes machinery) the sound gets ignored.
ALSO display the live log and you can watch what it is picking up in real time. If you see the owl being heard and identified, you know your setup is good. If then it is not being uploaded I still lean back to the species occurrence threshold. If the bird is not listed in your threshold it is ignored. JUST THINKING also, your location may mean the score for that bird is different than mine. For me the barred owl for eg is level   You can see yours by clicking on the MORE INFO link beside the frequency threshold and then click on the species list tester. For my latitude and longitude for the barred owl is shows: Strix varia_Barred Owl - 0.011556869

Maybe yours is lower and so you have to adjust the threshold number.

**Comment by Robtlord** [comment]
Again thanks for suggestions where to look ... I am learning the site pages well by now!    Strix varia_Barred Owl - 0.0033969285 for my area and I am set at 0.001.  Last night and this morning they were hooting like crazy and still no recording. I have played the Merlin recording as well , right beside the microphone... nothing. I am getting other birds though. (no Canadian Geese either). I'd hate to have to reinstall the program ... I will as a last resort.

**Comment by Robtlord** [comment]
Played the Merlin app calls for Barred owl on a few different days and all come up with a toad? 
<img width="880" alt="Screenshot 2023-09-06 at 9 28 03 AM" src="https://github.com/mcguirepr89/BirdNET-Pi/assets/142046430/d7b68863-d4fa-4d9b-a3d9-6cd0982dbda5">

**Reply by waynecarnold** [reply]
Looks like to me then your detections are getting distorted leading to wrong result. If you listen back to the spectrograph does it sound like the recording? Also those scores are too low if you played the birdsong right next to the microphone, they should be up around 0.6 -0.9 confidence scores. I would plug in mic directly to the pi and see if that changes anything, if not can you change the mic? Otherwise beyond me what to do next. Fyi my mic is a cheap $8 usb mic on a long usb extension 

![IMG_2440](https://github.com/mcguirepr89/BirdNET-Pi/assets/136210780/7bbcd4b6-b909-4638-9b64-9037b8d19d05)


![IMG_2439](https://github.com/mcguirepr89/BirdNET-Pi/assets/136210780/b9dc61ad-6587-42e2-aa7a-c9a00f6147ac)

**Reply by waynecarnold** [reply]
HAlso you may have to reduce the confidence temporarily to below what you are seeing in the log to get it to actually write the file so you can listen to the spectrogram by going to the “todays detections” area.

**Comment by Robtlord** [comment]
Tried different usb ports with and without the extension cord, the playback is mostly a hum, every 30 seconds it records and it is always the "Toad" ... if it weren't frustrating it would be funny! More than likely I probably have to start from scratch, uninstall the Birdnet-pi and re-install. I've gotta travel for a couple weeks and so will leave this to my return. I do want to thank you for taking your time offering help. I will get it right eventually.

**Comment by Robtlord** [comment]
Ok so I am am OCD but I decided to try one last time and so brought everything back into the house. I tested the Mic and also increased the gain in the Terminal alsamixer. Then got out both the Sibley's and Merlin apps and played from both 8 species and all came up except the barred owl (1  45% only) In the logs I watched it state "Human" for the owl. I played 8 different calls from 2 apps and all said the same "Human"  The threshold is at 0.001 ... so until Birdnet has a batter database on Barred Owl calls I'll just have to let this go!  I'll get it set up outside again and clear the logs tomorrow.   The last question might be how often does one update the Birdnet sound database?  Again thanks

**Comment by waynecarnold** [comment]
I use the system controls section under tools and if it shows an update then i do it, otherwise i have left everything as is.

**Comment by Robtlord** [comment]
I am getting recordings but it seems to be almost all hum. I have a 6 ft USB extension (mic info above) Thought I'd post a typical photo of the Spectrograph if that might trigger a suggestion for improvement ideas from folks out there!
<img width="1084" alt="Screenshot 2023-09-10 at 7 44 01 AM" src="https://github.com/mcguirepr89/BirdNET-Pi/assets/142046430/c5554abf-b03c-4580-b71a-e37c44d1652e">
